File: Modul 4/py/PRAK405-2410817310013-AhmadAlifPratamaPutra.py
data = input().split()
n = int(data[0])
kelipatan = int(data[1])

total = 0
for i in range(1, n + 1):
    baris = sum(j * kelipatan for j in range(1, i + 1))
    total += baris
    print(f"({' + '.join([f'{j} * {kelipatan}' for j in range(1, i + 1)])}) = {baris}")

print(total)

File: Modul 4/c/PRAK405-2410817310013-AhmadAlifPratamaPutra.c
#include <stdio.h>
int main() {
    int n, kelipatan, i, j, total = 0, baris;

    scanf("%d %d", &n, &kelipatan);

    for (i = 1; i <= n; i++) {
        baris = 0;
        printf("(");
        for (j = 1; j <= i; j++) {
            baris += j * kelipatan;
            printf("%d * %d", j, kelipatan);
            if (j < i) {
                printf(" + ");
            }
        }
        total += baris;
        printf(") = %d\n", baris);
    }
    printf("%d\n", total);

    return 0;
}